vc-verifier/kotlin/vcverifier/src/main/java/io/mosip/vercred/vcverifier/credentialverifier/validator/MsoMdocValidator.kt

```diff
@@ -30,7 +33,7 @@ class MsoMdocValidator {
             val validUntil: DataItem? = validityInfo["validUntil"]
             if (validUntil == null || validFrom == null) {
                 Log.e(tag, "validUntil / validFrom is not available in the credential's MSO")
-                throw StaleDataException("invalid validUntil / validFrom in the MSO of the credential")
+                throw ValidationException(ERROR_MESSAGE_INVALID_DATE_MSO, ERROR_CODE_INVALID_DATE_MSO)
```

Comment:
If we are now throing Validation exception instead of StaleDataException, can we remove StaleDataException if unused?